public/app/core/context/GrafanaContext.ts

```diff
@@ -31,5 +30,14 @@ export function useGrafana(): GrafanaContextType {
 // @grafana/runtime
 export function useReturnToPreviousInternal() {
   const { chrome } = useGrafana();
-  return chrome.setReturnToPrevious;
+  return useCallback(
+    (title: string, href?: string) => {
```

Comment:
this means we have a mismatch in the interface between what's defined here and in `packages/grafana-runtime/src/utils/returnToPrevious.ts`.

either it should be:
- 2 separate arguments, e.g. `(title: string, href?: string) => {`
- one argument with the options as an object, e.g. `(options: { title: string, href?: string }) => {`

i have a very slight preference for the first one, as otherwise a lot of people will end up passing an object with only one key, e.g. `{ title: 'whatever' }`

so that would mean leaving this line as is, but changing `packages/grafana-runtime/src/utils/returnToPrevious.ts` to have something like:
```tsx
type ReturnToPreviousHook = () => (title: string, href?: string) => void;
```